FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth/demo_config.h

```diff
@@ -143,6 +143,20 @@
  * #define democonfigCLIENT_PRIVATE_KEY_PEM    "...insert here..."
  */
 
+/**
+ * @brief The username value for authenticating client to the MQTT broker when
+ * username/password based client authentication is used.
+ *
```

Comment:
There probably should be information about when username/password is supported (including information saying that when username/password is provided, then client cert and private keys are not required).
Also, the link to the AWS docs for more information on custom authentication can be provided: https://docs.aws.amazon.com/iot/latest/developerguide/custom-authentication.html

Reply:
I've added a comment in case they decide to use AWS IoT. However, product 3 demos are for connecting to a local broker. I am including instructions for how to setup a local Mosquitto server to work with the TLS demos in a separate PR.